src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -240,15 +294,12 @@ def fall_detect(self, image=None):
                 log.info("No pose with key-points found.")
                 return inference_result, thumbnail
             else:
-                pose_dix = {}
                 inference_result = []
+                pose_score, pose_dix = self.estimate_spinalVector_score(pose)
```

Comment:
@bhavikapanara why are we calling `estimate_spinalVector_score` here? The spinal score was already calculated a few lines above in `pose, thumbnail = self.find_keypoints(image)`. Shouldn't we just return the score in `find_keypoints` and reuse it?

Reply:
yes..got it. Done